src/themes/five/src/components/OrderDetails/index.js

```diff
@@ -315,6 +315,12 @@ const OrderDetailsUI = (props) => {
                   <span>{order?.comment}</span>
                 </CommentContainer>
               )}
+              {[3, 4].includes(order?.delivery_type) && order?.place && (
```

Comment:
please don't use this validation because is hardcoded, create a constant for this